workers/Dockerfile.base

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:16.04
+
+RUN \
+  apt-get update -qq && \
+  apt-get install -y \
+                     lsb-release && \
+  echo "deb http://archive.ubuntu.com/ubuntu $(lsb_release -sc) multiverse\n" \
+      >> /etc/apt/sources.list.d/added_repos.list && \
+  echo "deb http://cran.cnr.berkeley.edu/bin/linux/ubuntu $(lsb_release -sc)/" \
+      >> /etc/apt/sources.list.d/added_repos.list && \
+  apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9 && \
+  apt-get update -qq && \
+  apt-get install -y \
+                     ed \
+                     git \
+		     mercurial \
+		     libcairo-dev \
+		     libedit-dev \
+                     lsb-release \
+		     python3 \
+		     python3-pip \
+		     r-base \
+		     r-base-dev \
+                     libxml2-dev \
+                     libcurl4-gnutls-dev \
+                     libssl-dev \
+                     curl \
+		     wget &&\
+  rm -rf /var/lib/apt/lists/*
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+ENV R_LIBS "/usr/local/lib/R/site-library"
```

Comment:
Gotcha. That makes sense. I was overlooking the `/usr/` part, I guess, and jumping straight to `site-library`.